controller/array_action/array_mediator_ds8k.py

```diff
@@ -292,11 +275,12 @@ def _extend_volume(self, api_volume, new_size_in_bytes):
         try:
             self.client.extend_volume(volume_id=api_volume.id,
                                       new_size_in_bytes=new_size_in_bytes)
-        except exceptions.NotFound:
-            raise array_errors.ObjectNotFoundError(api_volume.id)
-        except (exceptions.ClientError, exceptions.ClientException) as ex:
+        except exceptions.ClientException as ex:
```

Comment:
I'm not 100% like the new implementation. I think it's because the language supports having a couple of `except` statements for one `try` statement and we choose to do it in a different way. What's the reason for this change?

Reply:
that way we can catch everything we are looking for and reducing the chance of missing anything, plus, it's more "organized" this way.

Reply:
We catch "everything" (`ClientException` is everything in this case) anyway, but if @oriyarde and you like this way more than my suggestion, it's fine by me.

Reply:
https://youtu.be/ussCHoQttyQ

Reply:
done